workitem/typegroup.go

```diff
@@ -57,7 +57,7 @@ func TypeGroups() []WorkItemTypeGroup {
 			ID:     scenariosID,
 			Bucket: BucketPortfolio,
 			Name:   "Scenarios",
-			Icon:   "fa fa-suitcase",
+			Icon:   "fa fa-bullseye",
```

Comment:
This looks good but I'm trying to find out this should maybe also be the icon of a work item of type scenario. Currently a scenario has a bolt icon.

Reply:
I agree with @kwk here - why do the icons in the sidebar not match the icons of the work item type? If they are not supposed to, should they not share names?

Reply:
I too agree, there should NOT be two different icons for 'Sidebar' and 'WI type'. I will ask Sunil to update the VD screens tomorrow and Sukhvir will update based on visual design.